vendor/engines/secure_rooms/app/services/secure_rooms/access_handlers/order_handler.rb

```diff
@@ -17,29 +17,33 @@ def initialize(occupancy)
       def process
         return unless occupancy.account_id?
 
-        ActiveRecord::Base.transaction do
-          create_order
-          create_order_detail
-
-          order.validate_order!
-          order.purchase!
-        end
+        create_order unless occupancy.order_detail_id?
```

Comment:
if the `occupancy` here is `exit_at` only, we create the order for it, but it will fail the following condition and thus no fulfill attempt will be made--this should pave the way for either both-at-same-time and "scan in, scan in again and 2 orders are created"